src/AqueductV1Auction.sol

```diff
@@ -123,7 +124,13 @@ contract AqueductV1Auction is IAqueductV1Auction {
         uint256 bidValue = bid;
         (reserve0, reserve1, ) = IAqueductV1Pair(pair).getReserves();
         if (token == token1) {
-            bidValue = (bid * reserve0) / reserve1;
+            if (auction.token == token0) {
```

Comment:
Are we confident with this logic? Do you think we need a test case for all 4 branches?

Reply:
added auction:bid0,0 and auction:bid1,1 tests in 5b2512c, which now gives full branch coverage